rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/execution/MainExecutionContext.kt

```diff
@@ -122,9 +123,10 @@ object MainExecutionContext {
     }
 
     /**
-     * @return an instance of jariko configuration
+     * @return an instance of jariko configuration.
+     * First af all the configuration is searched in [SystemInterface].
```

Comment:
I think you meant "First of all" instead of "First af all"?

Reply:
Yes